target/linux/armsr/modules.mk

```diff
@@ -287,6 +287,23 @@ define KernelPackage/octeontx2-net
 endef
 $(eval $(call KernelPackage,octeontx2-net))
 
+define KernelPackage/renesas-net-avb
+SUBMENU:=$(NETWORK_DEVICES_MENU)
+TITLE:=Renesas network drivers
+DEPENDS:=@(TARGET_armsr_armv8) +kmod-phylink +kmod-mii +kmod-ptp +kmod-libphy +kmod-mdio-gpio
+KCONFIG:=CONFIG_NET_VENDOR_RENESAS=y \
```

Comment:
Selecting `CONFIG_NET_VENDOR_RENESAS` is not needed here, the generic configuration already activate it.

Reply:
Thanks. I've removed it here.